task/write-file/0.1/tests/run.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: write-file-pipeline
+spec:
+  workspaces:
+    - name: shared-workspace
+  tasks:
+    - name: output-credentials
+      taskRef:
+        name: write-file
+        kind: Task
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+      params:
+        - name: path
+          value: ./config/login.ini
+        - name: permissions
+          value: "0400"
```

Comment:
Waking up this morning, I realized that this should perhaps be `mode`? It seems Ansible, Puppet and Chef all call this parameter mode.